draft-gpew-priv-ppm.md

```diff
@@ -730,8 +730,8 @@ previous one (See {{anti-replay}}.)
 
 The response is an HTTP 200 OK with a body consisting of the helper's updated
 state and a sequence of *sub-responses*, where each sub-response corresponds to
-the sub-request in the same position in `AggregateReq`. The structure of each
-sub-response is specific to the PPM protocol:
+the sub-request in the same position in `AggregateReq`. Each sub-response
+encodes the timestamp and nonce, as well as a VDAF-specific `message`:
```

Comment:
```suggestion
encodes the timestamp, a nonce, and a VDAF-specific `message`:
```

Reply:
Took this but merged "timestamp" into "nonce".